docs/importing-js.md

```diff
@@ -26,12 +26,20 @@ of your built assets, you will want to explicitly reference the package's `index
 import {MDCFoo, MDCFooFoundation} from '@material/foo/index';
 ```
 
+Certain build tools will often pick up on the use of `package.json`'s `module` property, which points to the ES2015+ `index.js` source code. If you are using [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/guide/en), then you will not need to reference the `index.js` file, and can continue to use the `@material/foo` import paths.
```

Comment:
```markdown
Certain build tools will detect the `module` property in `package.json`,
which points to ES2015+ source code in an `index.js` file.
If you're using [Webpack](https://webpack.js.org/) or
[Rollup](https://rollupjs.org/guide/en), you do not need to
reference `/index` directly, and can continue to use the shorter
`@material/foo` import path syntax.
```

(Reformat the above into one line)

Reply:
It looks like this comment hasn't been applied.

Furthermore, index.js isn't ES2015+ source anymore. It's ES5 source in an ES module.

Reply:
How would you phrase "ES5 source in an ES module" in context?

Reply:
The 2 index.js files of the npm package are:

```js
dist/index.js (ES5) -- UMD
index.js (ES2015+) -- ESModule
```

Isn't what Andy wrote correct?

Reply:
The distinction is that prior to the TS conversion, *.js was ES modules _with ES2015+ syntax_, thus requiring babel and additionally the object assign polyfill. After the TS conversion, *.js are ES modules _with ES5 syntax_. I believe that means you can get away with just webpack and not babel at all?

In terms of phrasing, something like: "which points to an ES module otherwise containing only ES5 syntax". We could link something to Rollup's pkg.module definition at https://github.com/rollup/rollup/wiki/pkg.module if we want.

Reply:
@kfranqueiro I just checked by running `npm run build:esmodules`, so I do think you're correct.

Also I just noticed that we don't use `Object.assign` anywhere in our `packages/` directory. I do not think you need babel for either of our UMD or ESM packages.

Reply:
RE your last point, that's what I mean. Previously we were relying on the object assign polyfill to generate our UMD (you were never expected to need babel to consume those). Now we don't need to use that polyfill for our build process, and neither does anyone else using us within their own.

Reply:
In getting-started.md there is this dep:
```
- [babel-plugin-transform-object-assign](http://npmjs.org/package/babel-plugin-transform-object-assign), for IE 11 support
```

I am going to remove it